src/app/static/src/app/components/projects/ProjectHistory.vue

```diff
@@ -61,18 +61,19 @@
                 </button>
             </template>
         </modal>
-        <modal :open="projectToCopy || versionToCopy">
-            <h4 v-if="projectToCopy" v-translate="'copyProjectHeader'"></h4>
-            <h4 v-if="versionToCopy" v-translate="'copyVersionHeader'"></h4>
+        <modal :open="versionToCopy">
+            <h4 v-translate="'copyVersionHeader'"></h4>
```

Comment:
This header could be changed to give the actual version name once it is combined with Emma's ticket that is adding in version names. Something like `Copying ${versionName} to a new project`

Reply:
I have now merged in the latest master and added in this feature.

Reply:
Re translations, see this draft for an example of interpolating with `i18next`: https://github.com/mrc-ide/hint/pull/336/files#diff-305dda242c365630c350e025be821a5cR113